packaging/debian/xpra/control

```diff
@@ -214,6 +214,7 @@ Recommends: xpra-codecs (= ${binary:Version})
 # hope that this will allow our status icon to be shown:
         ,gir1.2-appindicator3-0.1
         ,gir1.2-ayatanaappindicator3-0.1
+        ,gnome-shell
```

Comment:
This is not acceptable, we cannot depend on `gnome-shell`.
Xpra is DE agnostic.
Ideally this would live in another package that users can choose to install to support gnome shell.
This may warrant adding a `xpra-client-gnome` package.

Reply:
The xpra-client-gnome DEB package is added similar to its RPM counterpart.